tools/validators/ontology_validator/yamlformat/validator/presubmit_validate_types_lib.py

```diff
@@ -118,6 +130,24 @@ def GetEntityType(self, namespace_name, typename):
       return None
     return self.entity_type_universe.GetEntityType(namespace_name, typename)
 
+  def GetUnitsMapByMeasurement(self, field_name):
```

Comment:
this should really be a method on the units universe.  Whether or not you want a proxy to it here is up to you.